test/config/mysql/mysql.sql

```diff
@@ -0,0 +1,3 @@
+GRANT ALL ON *.* TO 'sequelize_test'@'%' with grant option;
+
+FLUSH PRIVILEGES;
```

Comment:
I'm not sure if it would be better to add this file or just use the one for MariaDB since they are exactly the same.

Reply:
I think it's better to keep separate files.